README.md

```diff
@@ -45,198 +29,104 @@ The platform incorporates the following components:
+- <a id="real-time-user-segmentation-demo"></a>[**Real-time user segmentation**](demos/slots-stream/real-time-user-segmentation.ipynb) &mdash; builds a stream-event processor on a sliding time window for tagging and untagging users based on programmatic rules of user behavior.
 
-Data in the platform &mdash; including collected data, internal or external telemetry and logs, and program-output data &mdash; can be analyzed and visualized in different ways simultaneously.
-The platform supports multiple standard data analytics and visualization tools, including SQL, Prometheus, Grafana, and pandas.
-For example, you can plot or chart data within Jupyter Notebook using [Matplotlib](https://matplotlib.org/); use your favorite BI visualization tools, such as [Tableau](https://www.tableau.com), to query data in the platform over a Java database connectivity connector (JDBC); or build real-time dashboards in Grafana.
+<a id="additional-demos"></a>
+### Additional Demos
 
-The data analytics and visualization tools and services generate telemetry and log data that can be stored using the platform's time-series database (TSDB) service or by using external tools such as [Elasticsearch](https://www.elastic.co/products/elasticsearch).
-Platform users can easily instrument code and functions to collect various statistics or logs, and explore the collected data in real time.
+You can download additional demos from GitHub &mdash; for example:
 
-The [Grafana](https://grafana.com/grafana) open-source analytics and monitoring framework is natively integrated into the platform, allowing users to create dashboards that provide access to platform NoSQL tables and time-series databases from different dashboard widgets.
-You can also create Grafana dashboards programmatically (for example, from Jupyter Notebook) using wizard scripts.
-For information on how to create Grafana dashboards to monitor and visualize data in the platform, see [Adding a Custom Grafana Dashboard](https://www.iguazio.com/docs/tutorials/latest-release/getting-started/grafana-dashboards/).
+- <a id="xgboost-demo"></a>[**XGBoost classification**](https://github.com/mlrun/demo-xgb-project) &mdash; uses XGBoost to perform binary classification on the popular Iris ML data set, and runs parallel model training with hyperparameters.
+- <a id="image-classification-demo"></a>[**Image classification**](https://github.com/mlrun/demo-image-classification) &mdash; builds and trains an ML model that identifies (recognizes) and classifies (labels) images by using Keras, TensorFlow, and Horovod.
+- <a id="serverless-spark-demo"></a>[**Serverless Spark**](demos/spark/mlrun_sparkk8s.ipynb) &mdash; demonstrates how to run the same Spark job locally and as a distributed MLRun job over Kubernetes.
 
-<a id="end-to-end-use-case-applications"></a>
-## End-to-End Use-Case Applications
+For information on the available demos, contact the Iguazio [support team](mailto:support@iguazio.com).<br>
```

Comment:
We should point to a place we update the list of demos

Reply:
Yes, we should, once we have one ... . I added this as a temporary "solution" so that the doc would be accurate even before we decided on and implemented a shared demos support-matrix solution.

Reply:
we can use mlrun/demos, seems like the easiest place